crates/bevy_ecs/src/change_detection.rs

```diff
@@ -81,6 +62,46 @@ pub trait DetectChanges {
     /// [`SystemChangeTick`](crate::system::SystemChangeTick)
     /// [`SystemParam`](crate::system::SystemParam).
     fn last_changed(&self) -> u32;
+}
+
+/// Types that implement reliable change detection.
```

Comment:
This description feels too broad: this is only for the Mut form.

Reply:
All of this is unchanged, git just got confused because I put the new DetectChanges first.

Reply:
Ah, okay.